### ISSUE TITLE
Create Issue and Pull Request Templates

### ISSUE THREAD

**Issue by naveen8801** [CodeFlow201/codeflow#79]
Issue and PR templates are not present in this project. Templates help in better understanding of issues and PR.

I'd like to work on it as  LGM SOC participant, please assign it to me :-)

**Comment by Subhangini** [comment]
I would like to work on this

**Comment by hkaur008** [comment]
go on !

**Comment by Subhangini** [comment]
I am having my exams till 13 will it be okay if i generate a pr till 15.

**Comment by hkaur008** [comment]
Yes if anyone else, doesn't pick it till 15 . 😅. You can do it 😊.
